examples/express-ts/src/logger.ts

```diff
@@ -0,0 +1,32 @@
+import * as store from '@leapfrogtechnology/async-store';
+
+/**
+ * Get unique request id from store or return empty string.
+ *
+ * @returns {string}
+ */
+function getRequestId() {
+  return (store.getId() || '').substring(0, 8);
+}
+
+/**
+ * Write info logs and associated request id to stdout.
+ *
+ * @param {string} text
+ */
+export function info(text: string) {
+  const requestId = getRequestId();
+
+  process.stdout.write(`[ INFO ] ${requestId ? `[ ${requestId} ]` : ''} ${text}\n`);
```

Comment:
```diff
- process.stdout.write(`[ INFO ] ${requestId ? `[ ${requestId} ]` : ''} ${text}\n`);
+ process.stdout.write(`[ INFO ]   ${requestId ? `[ ${requestId} ]` : ''} ${text}\n`);
```

Reply:
Add an extra space after `[ INFO ]` so that the request id are aligned.

```
[ DEBUG ] [ 006a0e32 ] Received b: 30
[ INFO ]  [ 006a0e32 ] Simulating Delay
```

Reply:
I added a space after `INFO` to maintain the alignment. Is it okay??